CODEOWNERS

```diff
@@ -53,9 +53,9 @@ go.mod @fleetdm/go
 # (1 or more infra-literate engineers is required to review changes.)
 # FUTURE: Look for a way to not have this notify every single person in this "github team".
 ##############################################################################################
-/infrastructure/ @fleetdm/infra @edwardsb
-/charts/ @fleetdm/infra @edwardsb
-/terraform/ @fleetdm/infra @edwardsb
+/infrastructure/ @rfairburn @ksatter @lukeheath
+/charts/ @rfairburn @ksatter @lukeheath
+/terraform/ @rfairburn @ksatter @lukeheath
```

Comment:
@Sampfluger88 Can we put Ben back here? Kathy is going to be on PTO, and we need Ben to approve Robert's PRs.

Reply:
Yes, that error was happening because Ben was removed from the GitHub org. I assumed that was unintentional so I added him back.